### PR TITLE
Fixed hardcode

### PR DESCRIPTION
**Pull request by jf17**
Improved work with the translation of the game.

Fixed  issue #35

### CODE REVIEW

File: GameEngine/Lang/en.php [hunk 1]
```diff
@@ -130,6 +130,7 @@
 define("CROP_COM",CROP." consumption");
 define("PER_HR","per hour");
 define("PROD_HEADER","Production");
+define("CAPITAL_NAME","Capital");
```
Comment (andreapavoni) [inline GameEngine/Lang/en.php]: I'd use the name `CAPITAL` rather than `CAPITAL_NAME` which makes think about the name the player used for their capital.

Comment (jf17) [reply GameEngine/Lang/en.php]: The name `CAPITAL` is already being used! 
`define("CAPITAL","This is your capital");`

Comment (jf17) [reply GameEngine/Lang/en.php]: @andreapavoni 
Is it possible to use a different variable name? 
For example `CAPITAL_HEADER` ?

Comment (jf17) [reply GameEngine/Lang/en.php]: Or 
`CAPITAL_TITLE` ,
`CAPITAL_LABEL` ?

Comment (andreapavoni) [reply GameEngine/Lang/en.php]: I'd go with `CAPITAL_LABEL`, sounds more appropriate

Comment (jf17) [reply GameEngine/Lang/en.php]: @andreapavoni 
Sent a new commit .

**Pull request merged**